helpers/lib_tasks_docker_release.py

```diff
@@ -62,6 +62,251 @@ def _get_dev_version(version: str, container_dir_name: str) -> str:
     return dev_version
 
 
+def _create_multiarch_builder(
+    ctx: Any,
+) -> None:
+    """
+    Create a multi-arch builder for Docker buildx.
+
+    :param ctx: invoke context
+    """
+    # Create a multi-arch builder.
+    platform_builder_name = "multiarch_builder"
+    cmd = rf"""
+    docker buildx rm {platform_builder_name}
+    """
+    # We do not abort on error since the platform builder might be present
+    # or not from previous executions.
+    hsystem.system(cmd, abort_on_error=False)
+    cmd = rf"""
+    docker buildx create \
+        --name {platform_builder_name} \
+        --driver docker-container \
+        --bootstrap \
+        && \
+        docker buildx use {platform_builder_name}
+    """
+    hlitauti.run(ctx, cmd)
+
+
+def _build_multi_arch_image(
+    ctx: Any,
+    opts: str,
+    multi_arch: str,
+    build_args: str,
+    build_image: str,
+    dockerfile: str,
+) -> None:
+    """
+    Build a multi-architecture Docker image.
```

Comment:
Let's rephrase it to mention that the image is being built in a remote Docker registry.